src/jagadeeshMJun20/FibonacciSeries.java

```diff
@@ -0,0 +1,47 @@
+/*Program 2: To print first n Fibonacci Numbers.
+Input n = 5
+Output = 1 1 2  3  5  8 
+
+Input: n = 10 
+Output :  1 1 2  3  5  8 13  21 34 55
+
+Hint :
+The next number is found by adding up the two numbers before it:
+the 2 is found by adding the two numbers before it (1+1),
+the 3 is found by adding the two numbers before it (1+2),
+the 5 is (2+3),
+and so on! */
+package jagadeeshMJun20;
+
+import java.util.Scanner;
+
+public class FibonacciSeries {
+	private int num;
```

Comment:
Nicely used private variable and method.

Reply:
Yes, concept of encapsulation.